src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/yflows/YFlowCreateSpec.groovy

```diff
@@ -440,6 +439,44 @@ source: switchId="${flow.sharedEndpoint.switchId}" port=${flow.sharedEndpoint.po
         lagPort && northboundV2.deleteLagLogicalPort(swT.shared.dpId, lagPort)
     }
 
+    @Tidy
+    @Tags([TOPOLOGY_DEPENDENT, LOW_PRIORITY])
+    def "System allows to create y-flow with bandwidth equal to link bandwidth between shared endpoint and y-point (#4965)"() {
+        /* Shared <----------------> Y-Point ---- Ep1
+                          |              \ _______ Ep2
+           flow bw == bw of this link   flow bw <= bw on these two links
+         */
+
```

Comment:
Could you please explain a bit more which bw is the total bandwidth of the isl and which is the max bandwidth of the request?

Reply:
Do you propose to add link to youtube video, where I explain this test on a whiteboard for 10 minutes? %)
I guess 'bandwidth equal to link bandwidth between shared endpoint and y-point' is clear enough itself.

Reply:
Actually, in the request, you don't specify bandwidth, but max_bandwidth. I propose something like this.

```
"System allows to create y-flow with max_bandwidth equal to link bandwidth between shared endpoint and y-point."

        /* Shared <----------------> Y-Point ---- Ep1
                          |              \ _______ Ep2
           flow max_bandwidth == bw of this link   flow max_bandwidth <= bw on these two links
         */
```

Reply:
Fixed and improved with emojis!

Reply:
💃